contracts/LiquidPledgingBase.sol

```diff
@@ -64,7 +66,7 @@ contract LiquidPledgingBase is Owned {
         uint64 intendedProject; // Used when delegates are sending to projects
         uint64 commitTime;  // When the intendedProject will become the owner  
         uint64 oldPledge; // Points to the id that this Pledge was derived from
-        PaymentState paymentState; //  Pledged, Paying, Paid 
+        PaymentState paymentState; //  Pledged, Paying, Paid //TODO Very confusing with PaymentStatus rename to pledgeState??????????????
```

Comment:
I agree

Reply:
Sweet, I don't know if it needs to delay the launch but i would like to leave the comments in and at least make it part of the next develop branch